tests/integ/test_stored_procedure.py

```diff
@@ -1027,7 +1028,15 @@ def return1(session_: Session) -> str:
         "runtime_version",
         "packages",
         "installed_packages",
+        # This seems like an unintended change from the server, we should remove it once it is removed from server
+        "is_aggregate",
```

Comment:
I had a chat some time back with @sfc-gh-fdrozdowski and I recall it being an intentional change. Can you confirm @sfc-gh-fdrozdowski

Reply:
IIUC, this field should not be shown for stored proc as it doesn't make sense, see https://snowflake.slack.com/archives/C04A6RQ3571/p1689185938064419?thread_ts=1689182211.748549&cid=C04A6RQ3571. @sfc-gh-ddalmeida Do you have a jira to track remove `is_aggregate` for stored procs?

Reply:
It is probably intentional for UDF, but should not be intentional for stored proc. I also chatted with @sfc-gh-ddalmeida on this. It should be easy to hide this property for stored proc.